src/components/input-dropdown/input-dropdown.component.ts

```diff
@@ -74,6 +74,8 @@ export class InputDropdownComponent<T> implements ControlValueAccessor, AfterVie
     /** Unsubscribe from all observables on component destroy */
     private readonly _onDestroy$ = new Subject<void>();
 
+    constructor(private readonly _changeDetector: ChangeDetectorRef ) { }
```

Comment:
Only a minor formatting comment but remove the space after ChangeDetectorRef - this seems to be in a lot of other files too.